wazuh/wazuh-kibana-app
# Feature Disable Wazuh by roles

Hi team!

this adds the feature to set roles in `wazuh.yml` to disable Wazuh App.

Closes #3164 

Test case:

- Add disabled role entry in `wazuh.yml`
```
disabled_roles: 
   - wazuh_disabled
```
![image](https://user-images.githubusercontent.com/9343732/117034121-5e9a8e00-acd9-11eb-88c0-14253717ab8d.png)


- Make sure the logged in user has the role assigned
![image](https://user-images.githubusercontent.com/9343732/117033856-2bf09580-acd9-11eb-94b6-389c20d78c02.png)


- Wazuh should be disabled when logging in or refreshing the browser
![image](https://user-images.githubusercontent.com/9343732/117033968-3f9bfc00-acd9-11eb-8bab-4d869bc68a56.png)

![image](https://user-images.githubusercontent.com/9343732/117034043-4d518180-acd9-11eb-8d23-2eae572b097c.png)
